config/plugin-site-issues.yaml

```diff
@@ -0,0 +1,17 @@
+ingress:
+  enabled: true
+  className: public-nginx
+  annotations:
+    "cert-manager.io/cluster-issuer": "letsencrypt-prod"
+    "nginx.ingress.kubernetes.io/ssl-redirect": "true"
+    "nginx.ingress.kubernetes.io/enable-cors": "true"
+    "nginx.ingress.kubernetes.io/cors-allow-methods": "GET, OPTIONS"
+    "nginx.ingress.kubernetes.io/cors-allow-origin": "*"
+  hosts:
+    - host: plugin-site-issues.origin.jenkins.io
+    - host: plugin-site-issues.jenkins.io
```

Comment:
```suggestion
    - host: plugin-site-issues.origin.jenkins.io
      paths:
        - path: /
          pathType: Prefix
    - host: plugin-site-issues.jenkins.io
      paths:
        - path: /
          pathType: Prefix
```

Reply:
I suspect strongly we won't even need origin, but sounds good